laia/decoders/ctc_language_decoder.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List
+
+import numpy as np
+import torch
+from torchaudio.models.decoder import ctc_decoder
+
+from laia.losses.ctc_loss import transform_batch
+
+
+class CTCLanguageDecoder:
+    def __init__(
+        self,
+        language_model_path: str,
+        lexicon_path: str,
+        tokens_path: str,
+        language_model_weight: float = 1.0,
+        blank_token: str = "<ctc>",
```

Comment:
Don't we have these tokens in constants somewhere else ?

Reply:
These params are set by the user using `--decode.input_space`, `--decode.unk_token`, `--decode.blank_token`.